cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -186,12 +109,12 @@ func PerformAddonsUpgrade(client clientset.Interface, cfg *kubeadmapi.InitConfig
 	return errorsutil.NewAggregate(errs)
 }
 
-// unupgradedControlPlaneInstances returns a list of control palne instances that have not yet been upgraded.
+// UnupgradedControlPlaneInstances returns a list of control palne instances that have not yet been upgraded.
 //
 // NB. This function can only be called after the current control plane instance has been upgraded already.
 // Because it determines whether the other control plane instances have been upgraded by checking whether
 // the kube-apiserver image of other control plane instance is the same as that of this instance.
-func unupgradedControlPlaneInstances(client clientset.Interface, nodeName string) ([]string, error) {
+func UnupgradedControlPlaneInstances(client clientset.Interface, nodeName string) ([]string, error) {
```

Comment:
i think maybe this function belongs to a different file, but fine with me to keep it here too.